photoeditor/src/main/java/ja/burhanrashid52/photoeditor/PhotoEditorView.java

```diff
@@ -89,7 +89,7 @@ private void init(@Nullable AttributeSet attrs) {
         brushParam.addRule(RelativeLayout.ALIGN_BOTTOM, imgSrcId);
 
         //Setup GLSurface attributes
-        mImageFilterView = new ImageFilterView(getContext());
+        mImageFilterView = new ZedgeImageFilterView(getContext());
```

Comment:
If it is not too difficult I would first try to move this to a subclass of PhotoEditorView (like ZedgePhotoEditorView) that overrides the constructor to instantiate a ZedgeImageFilterView. It is too much of  a hassle leave it like this.